documentation/modules/selecting-migration-network-for-virt-provider.adoc

```diff
@@ -27,3 +27,21 @@ When the *Providers detail* page opens:
 . Click the *Networks* tab.
 . Click *Set default transfer network*.
 . Select a default transfer network from the list and click *Save*.
+. Configure a gateway in the network used for {project-short} migrations by completing the following steps:
+.. In the {ocp} web console, click *Networking* > *NetworkAttachmentDefinitions*.
+.. Select the appropriate NAD.
```

Comment:
```suggestion
. Select the appropriate selected default transfer network NAD.
```

Reply:
@sgratch I added "default transfer network," but we don't need "selected" after "appropriate".  
Lines 31-34 and line 47 are sub-steps of line 30, so I left the hierarchy alone.